packages/cli/src/commands/index.ts

```diff
@@ -72,6 +72,17 @@ export function registerRepoCommand(program: Command) {
     .action(
       lazy(() => import('./repo/list-deprecations').then(m => m.command)),
     );
+
+  command
+    .command('test')
+    .allowUnknownOption(true) // Allows the command to run, but we still need to parse raw args
+    .option(
+      '--since <ref>',
+      'Only test packages that changed since the specified ref',
+    )
+    .helpOption(', --backstage-cli-help') // Let Jest handle help
```

Comment:
It does if you use `--backstage-cli-help`. I flipped it around now though, so that `--help` displays these, and `--jest-help` displays the jest help options.